lib/meilisearch-rails.rb

```diff
@@ -611,14 +611,12 @@ def ms_init_raw_answer(json)
 
       def ms_search(query, params = {})
         if MeiliSearch::Rails.configuration[:pagination_backend]
-          page = params[:page].nil? ? params[:page] : params[:page].to_i
-          hits_per_page = params[:hitsPerPage].nil? ? params[:hitsPerPage] : params[:hitsPerPage].to_i
-          hits_per_page ||= params[:hits_per_page].nil? ? params[:hits_per_page] : params[:hits_per_page].to_i
-
           %i[page hitsPerPage hits_per_page].each do |key|
-            params[key] = params[key].to_i if params.key?(key)
+            params[key.to_s.underscore.to_sym] = params[key].to_i if params.key?(key)
           end
 
+          # to activate the finite pagination in Meilisearch it is required to have at least
+          # `hits_per_page` defined or `page` in the search request.
```

Comment:
Do you mean "to use Meilisearch v0.30.0 and later"? The users do not need to activate anything on their side right?

Reply:
Yeah, they don't! 

This comment aims to explain why we have a lose `params[:page] ||= 1` here. The purpose here is to ensure when the user adopts a `pagination_backend` we will use our finite pagination data, instead of the estimated one.